.github/workflows/release.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
           node-version: ${{matrix.nodeversion}}
           registry-url: ${{env.NPM_REGISTRY_URL}}
       - name: Setup Python
-        uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435 # tag=v4.2.0
+        uses: actions/setup-python@57ded4d7d5e986d7296eab16560982c6dd7c923b # tag=v4.2.0
```

Comment:
@MatanHeledPort: maybe change the tag from v4.2.0 to v4.6.0 so the next time dependabot updates this too!